examples/bzlmod/MODULE.bazel

```diff
@@ -102,7 +102,13 @@ pip.parse(
     python_version = "3.9",
     requirements_lock = "//:requirements_lock_3_9.txt",
     requirements_windows = "//:requirements_windows_3_9.txt",
-    target_platforms = ["all"],
+    # You can use one of the values below to specify the target platform
+    # to generate the dependency graph for.
+    target_platforms = [
```

Comment:
(Out of scope for this PR, just thinking out loud)

Hm, if we had a label_keyed_string_dict, then I think we could create an arbitrary mapping of conditions to platform strings? So if you had some exotic platform, you could do...

```
target_platform_mapping = {
  "@//myplatforms:chromeos_arm": "chromeos_arm"
}
```

And then, if we parse out "chromeos_arm" from the wheel name, we're able to put the corresponding label for the select.

Oh hm. Or maybe have a register() tag part of a bzlmod extension? e.g. `pip.register_platform(name="chromeos_arm", label="@//myplatforms:chromeos_arm", os="chromeos", arch="arm")`  ? The latter sounds like it would be useful for...something.

Reply:
I'll create a followup ticket.